Spanner/src/Transaction.php

```diff
@@ -312,7 +312,9 @@ public function delete($table, KeySet $keySet)
      *
      * Data Manipulation Language (DML) allows you to execute statements which
      * modify the state of the database (i.e. inserting, updating or deleting
-     * rows).
+     * rows). For example, DML supports INSERT, UPDATE and DELETE statements. For
+     * more on DML syntax, visit the DML syntax guide.
+     * {@see https://cloud.google.com/spanner/docs/dml-syntax}
```

Comment:
Move this down near line 363. URLs in the doc text should be formatted as markdown.